src/evagram/database/input_tool.py

```diff
@@ -0,0 +1,174 @@
+import pickle
```

Comment:
Can you change the directory structure to something like `src/evagram_input/input_tool.py`. Because input_tool is a script, I'm not sure we need another directory.